### PR TITLE
BAH-2186 | add sms service to docker compose

### PR DESCRIPTION
**Pull request by kavitha-sundararajan**

### CODE REVIEW

File: bahmni-lite/docker-compose.yml [hunk 1]
```diff
@@ -279,6 +279,12 @@ services:
     # volumes:
     # - "${APPOINTMENTS_PATH:?}/dist/:/usr/local/apache2/htdocs/appointments"
 
+  sms-service:
+    image: bahmni/sms-service:${SMS_SERVICE_IMAGE_TAG:?}
+    profiles: [ "emr" ]
```
Comment (mohan-13) [inline bahmni-lite/docker-compose.yml:284]: Lets not put this under `emr` profile as this profile should be the bare minimum set of services used to explore/try emr with lowest resources. We can create a new profile `sms` and document it on the steps to enable it

**Review by mohan-13 — COMMENTED** [review]
These changes would also be needed on bahmni-standard

File: bahmni-lite/docker-compose.yml (hunk 1 above)
Comment (angshu) [reply bahmni-lite/docker-compose.yml:284]: agree

Comment (kavitha-sundararajan) [reply bahmni-lite/docker-compose.yml]: updated profile

File: bahmni-lite/docker-compose.yml [hunk 2]
```diff
@@ -279,6 +279,12 @@ services:
     # volumes:
     # - "${APPOINTMENTS_PATH:?}/dist/:/usr/local/apache2/htdocs/appointments"
 
+  sms-service:
+    image: bahmni/sms-service:${SMS_SERVICE_IMAGE_TAG:?}
+    profiles: ["sms"]
+    environment:
+      SMS_TOKEN: ${SMS_TOKEN:?}
```
Comment (mohan-13) [inline bahmni-lite/docker-compose.yml:286]: @kavitha-sundararajan Add the environment variable for ORIGINATOR as well

Comment (kavitha-sundararajan) [reply bahmni-lite/docker-compose.yml:286]: added

**Review by mohan-13 — APPROVED** [review]

**Pull request merged**